Netwrix/netwrix_auditor/ingest/parser.yml

```
@@ -0,0 +1,90 @@
+name: netwrix_auditor_parser
+pipeline:
+  - name: cef_parser
+    description: ''
+    filter: ''
+    external:
+      name: cef.parse-cef
+      properties:
+        input_field: '{{original.message}}'
+        output_field: message
+  - name: general-setter
+    description: ''
+    filter: ''
+    external: null
+  - name: user-email-setter
+    description: ''
+    filter: '{{ "@" in cef_parser.message.suser }}'
+    external: null
+  - name: user-name-domain-setter
+    description: ''
+    filter: '{{ "\\" in cef_parser.message.suser }}'
+    external: null
+  - name: user-only-setter
+    description: ''
+    filter: '{{ "\\" not in cef_parser.message.suser and "@" not in cef_parser.message.suser }}'
+  - name: document-event-setter
+    description: ''
+    filter: '{{ cef_parser.message.cat == "Document" }}'
+    external: null
+  - name: logon-event-setter
+    description: ''
+    filter: '{{ cef_parser.message.DeviceProduct == "Logon Activity" }}'
+    external: null
+  - name: exchange-event-setter
+    description: ''
+    filter: '{{ cef_parser.message.DeviceProduct == "Exchange Online" }}'
+    external: null
+stages:
+  general-setter:
+    actions:
+      - set:
+          event.kind: 'event'
+          event.code: '{{ cef_parser.message.DeviceEventClassID |lower }}'
+          event.severity: '{{ cef_parser.message.Severity }}'
+          event.start: '{{ cef_parser.message.start }}'
+          observer.vendor: '{{ cef_parser.message.DeviceVendor}}'
+          observer.product: '{{ cef_parser.message.DeviceProduct}}'
+          observer.version: '{{ cef_parser.message.DeviceVersion}}'
+        filter: ''
+        name: set
+  document-event-setter:
+    actions:
+      - set:
+          file.path: '{{ cef_parser.message.filePath.split("/")[3:] |join("/") }}'
```

You can also provide `file.name` and `file.directory`:
```suggestion
          file.path: '{{ cef_parser.message.filePath.split("/")[3:] |join("/") }}'
          file.directory: '{{ cef_parser.message.filePath.split("/")[3:] |join("/") | dirname }}'
          file.name: '{{ cef_parser.message.filePath | basename }}'
```